lib/fund.js

```diff
@@ -14,13 +14,14 @@ const readShrinkwrap = require('./install/read-shrinkwrap.js')
 const mutateIntoLogicalTree = require('./install/mutate-into-logical-tree.js')
 const output = require('./utils/output.js')
 const openUrl = require('./utils/open-url.js')
-const { getFundingInfo, retrieveFunding, validFundingUrl } = require('./utils/funding.js')
+const { getFundingInfo, retrieveFunding, validFundingField, flatCacheSymbol } = require('./utils/funding.js')
 
 const FundConfig = figgyPudding({
   browser: {}, // used by ./utils/open-url
   global: {},
   json: {},
-  unicode: {}
+  unicode: {},
+  which: {}
```

Comment:
it would be nice to add a mention about `which` in `docs/content/cli-commands/npm-fund.md`

Reply:
Done! Feel free to reword as needed.